src/core/openthread-core-windows-config.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Should I put it in `./include` with the other config file or `./examples/driver/windows/include`?